python/dask_cudf/dask_cudf/backends.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
 
+from dask.dataframe.categorical import categorical_dtype_dispatch
```

Comment:
We need to bump the required version of dask in the dask-cudf `meta.yaml` as well as our `environment.yml` files for this 😄

Reply:
Bumped `dask` & `distributed` to `2.15.0` based on release post: https://github.com/dask/community/issues/45

@quasiben do let me know if this is fine?

(CI will error until there is a conda package available, I'll rerun tests once dask 2.15.0 is released today)

Reply:
2.15.0 is out now.  Thank you for bumping